composer.json

```diff
@@ -20,7 +20,7 @@
     },
     "require-dev": {
         "doctrine/phpcr-odm": "^1.4",
-        "symfony-cmf/testing": "^2.1@dev",
+        "symfony-cmf/testing": "dev-be_aware_of_deprecations",
```

Comment:
we need to merge the testing branch and update here before we can merge this PR

Reply:
yes, and the one on doctrine-phpcr too and maybe release it on doctrine or go on a @dev in testing